cypress/integration/table_tool_spec.js

```diff
@@ -79,14 +79,27 @@ context('Table Tool Tile',function(){
     });
     describe('edit table entries', function(){
         it('will add content to table', function(){
-            tableToolTile.getTableCell().first().type('0');
-            tableToolTile.getTableCell().last().type('5{enter}');
-            tableToolTile.getTableCell().first().should('contain','0');
-            tableToolTile.getTableCell().eq(1).should('contain', '5');
-            tableToolTile.getTableRow().should('have.length',2);
+            tableToolTile.getTableCell().first().type('3');
+            tableToolTile.getTableCell().last().click();
+            cy.wait(100);
+            tableToolTile.getTableCell().first().should('contain','3');
+
+            tableToolTile.getTableCell().last().type('2');
+            tableToolTile.getTableCell().first().click();
+            cy.wait(100);
+            tableToolTile.getTableCell().eq(1).should('contain', '2');
+
+            tableToolTile.getTableCell().first().type('1');
+            tableToolTile.getTableCell().last().click();
+            cy.wait(100);
+            tableToolTile.getTableCell().first().should('contain','1');
+
+            tableToolTile.getTableRow().should('have.length', 3);
+
             //also verify that new row is added when row "enter" key is sent to the last row
-            tableToolTile.getTableCell().first().type('{enter}');
-            tableToolTile.getTableRow().should('have.length',3);
+            // tableToolTile.getTableCell().last().type('{enter}');
```

Comment:
Shame, ever figure out why the tests don't work with `enter`?

Reply:
No, there was no evidence that the `{enter}` had any effect after my changes, so I just punted since I'd already spent more time on it than was probably warranted.